package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "apps",
+  "private": true,
   "scripts": {
-    "lint": "lerna run lint --concurrency=1",
-    "test": "lerna run test:ci --concurrency=1",
-    "bootstrap": "lerna bootstrap --no-ci --concurrency=1",
-    "deploy": "lerna run deploy",
-    "postinstall": "npm run bootstrap"
+    "bootstrap": "lerna bootstrap --no-ci",
+    "bootstrap:ci": "lerna bootstrap --ci --concurrency=2",
```

Comment:
I've disabled lerna concurrency due to CI flakiness for both test and bootstrap. Couldn't find much information on that - any experience with this?

Reply:
So I did some digging and:

- Using `npm ci` (so `lerna --ci`) is faster and more lightweight (according to npm: It can be significantly faster than a regular npm install by skipping certain user-oriented features) - never experienced any crashes/hanging with `--ci`
- Linting, testing and deployment can also be reliably run with concurrency of 2. 2 matches number of vCPUs we get in Circle
- The only problematic step is the build step. I don't know if it's problem with parcel (cache interference?) or it's so CPU-bound that it has hard time running in parallel but this is the only step that requires concurrency of 1 for stability

In the previous model building artifacts was part of test and deploy steps (sometimes repeated). Now we build all packages and then run tests on them and deploy in separate steps.

To sum up we build in series and everything else is done with parallelism of 2.

Reply:
I was running into this with bootstrap: https://github.com/lerna/lerna/issues/1457